File: oop/class_1.cpp
#include<bits/stdc++.h>
using namespace std;

class Room {
    public:
        int length;
        int  width;
        int height;   

        int area(){   
            return length * width;
        }

        double volume(){   
            return length * width * height;
        }

};

int main()
{
    Room r1;
    r1.height=4;
    r1.width=5;
    r1.height=6;
    cout<<r1.area()<<endl;
    cout<<r1.volume()<<endl;
}

File: oop/class_1.py
class Room:
   
    length = 0
    width = 0
    height = 0

    def area(self):
        return self.length * self.width

    def volume(self):
        return self.length * self.width * self.height


r1 = Room()
r1.height = 4
r1.width = 5
r1.length = 6
print(r1.area())
print(r1.volume())
